src/api/ledger/pathfind.js

```diff
@@ -46,8 +46,12 @@ function requestPathFind(remote, pathfind: PathFind, callback) {
       _.omit(utils.common.toRippledAmount(amount), 'value'));
   }
 
-  remote.requestRipplePathFind(params,
-    composeAsync(_.partial(addParams, params), callback));
+  remote.createPathFind(params, function(data) {
+    if (data.full_reply) {
```

Comment:
Passing a callback as last argument which is meant to be called multiple times is... not a common pattern, I think. I'd assume if the last argument to createPathFind is a function, that the user doesn't want to streams... also, first argument to the callback should be an error or null